### PR TITLE
feat: implement logic and api description for read public AccountV2 APIs

### PR DESCRIPTION
**Pull request by hvn2k1**
Part of https://github.com/bucketeer-io/bucketeer/issues/904

- Init route and API docs for AccountV2 public API
- Implement read only APIs for AccountV2:
  - GetAccountV2
  - GetAccountV2ByEnvironment
  - GetMe
  - ListAccounts
- Implement unit test suite

### CODE REVIEW

File: proto/gateway/service.proto [hunk 1]
```diff
@@ -577,4 +624,81 @@ service Gateway {
...
+          description: "The API Key generated on the Admin Console."
+          type: STRING,
+          required: true
+        }
+      }
+    };
+  }
+  rpc GetMe(GetMeRequest) returns (GetMeResponse) {
+    option (google.api.http) = {
+      get: "/v1/account/me"
```
Comment (cre8ivejp) [inline proto/gateway/service.proto:668]: Same here.

```suggestion
      get: "/v1/account/me/{organization_id}"
```

File: proto/gateway/service.proto [hunk 2]
```diff
@@ -577,4 +624,81 @@ service Gateway {
       }
     };
   }
+  rpc GetAccountV2(GetAccountV2Request) returns (GetAccountV2Response) {
+    option (google.api.http) = {
+      get: "/v1/account"
```
[the remark this answers is not in the document]
Comment (hvn2k1) [reply proto/gateway/service.proto]: I have moved param to path

File: proto/gateway/service.proto [hunk 3]
```diff
@@ -577,4 +624,81 @@ service Gateway {
...
+          type: STRING,
+          required: true
+        }
+      }
+    };
+  }
+  rpc GetAccountV2ByEnvironmentID(GetAccountV2ByEnvironmentIDRequest)
+      returns (GetAccountV2ByEnvironmentIDResponse) {
+    option (google.api.http) = {
+      get: "/v1/get-account-by-environment"
```
[the remark this answers is not in the document]
Comment (hvn2k1) [reply proto/gateway/service.proto]: I have updated the path

**Review by cre8ivejp — APPROVED** [review]
Nice work!
Thank you!

**Pull request merged**